backend/Makefile

```
@@ -121,7 +121,13 @@ index-data: ## Index data for Algolia search
 	@CMD="python manage.py algolia_update_synonyms" $(MAKE) exec-backend-command
 
 fetch-nest-dump: ## Download backend/data/nest.dump from S3 if the remote copy changed
```

checkmake (0.3.2)

[warning] Target body for "fetch-nest-dump" exceeds allowed length of 5 lines (7).

(maxbodylength)